frontend/ui/elements/screen_rotation_menu_table.lua

```diff
@@ -60,6 +60,44 @@ When unchecked, the default rotation of the file browser and the default/saved r
             separator = true,
         })
 
+        local function image_rotation_item(setting, label, separator)
+            return {
+                text = label,
+                radio = true,
+                separator = separator or false,
+                checked_func = function()
+                    return (G_reader_settings:readSetting("image_clockwise_rotation", true) == setting)
```

Comment:
(no need for parens :)
Maybe name the setting: `imageviewer_rotation_clockwise` (widget_effect_valueAffectedByTheBoolean) ?